Adding FAQ and Issue Templates

Durbatuluk1701 opened the pull request:
Inspired by the request for documentation in #1119, I have tried to create a Frequently Asked Questions (FAQ) markdown file.

I tried to only write answers to FAQ questions that I knew, but **I would definitely appreciate a knowledgeable eye to read over it and make sure I didn't get anything wrong**.

Along with the FAQ, I made some issue templates for Bug Reports and Feature Requests that I thought might be useful.

Durbatuluk1701 commented on .github/ISSUE_TEMPLATE/bug_report.md:
```diff
@@ -0,0 +1,59 @@
+**Environment (please complete the following information):**
+
+- **VsRocq Version:** [e.g., 2.2.5] (You can find this in the VS Code Extensions view)
+- **VsRocq Language Server Version (vscoqtop):** [e.g., 2.2.5] (Run `vscoqtop -v` or check `opam list`)
```
I think that's a great idea. Do we want to try to add some of this other info into it as well? 

It seems like right now it includes:
- Rocq Version + Ocaml version + Path
- Vscoqtop version + path

I think we could probably easily VS Code Version + VS Code Extension version, maybe OS as well?

rtetley replied:
Definitely ! VsCode version is easy, OS I can use https://nodejs.org/api/os.html but it's not as precise, still a good approx (we can now if it's linux, mac or windows). See #1124

Durbatuluk1701 commented on docs/FAQ.md:
```diff
@@ -0,0 +1,147 @@
+
+    - **VS Code Extension:**
+
+      Search for "vscoq" (by `maximedenes.vscoq`) in the VS Code Marketplace or VSCodium's Open VSX Registry and install it.
```
Okay I had thought the name change had been discussed but wasn't sure. I will make this change

Durbatuluk1701 commented:
> I am not sure how to handle the imminent renaming. Should we be pro-active and just change everything to vsrocq ? Or rather duplicate as some people will probably still be using vscoq ?

I went ahead and pro-actively renamed things, hopefully it seems reasonable.

rtetley commented on docs/FAQ.md:
```diff
@@ -0,0 +1,155 @@
+
+1.  **The Proof View / Goal Panel isn't showing.**
+
+    - Ensure VsRocq is installed correctly and "VsRocq: Path" points to `vscoqtop`.
```
```suggestion
    - Ensure VsRocq is installed correctly and "VsRocq: Path" points to `vsrocqtop`.
```

rtetley reviewed (commented):
Just the two vscotops still hanging there, otherwise good to merge !

Durbatuluk1701 commented:
> Ah I messed up, didn't see you already made the modifs to extension.ts.... I rebased your branch and corrected the typos, if you'll allow me to do a force push, I will push and merge !

Sure go ahead, thanks for the help!
